README.rst

```diff
@@ -10,18 +11,23 @@ Tutor plugin that enables execution of untrusted code in secure sandboxes using
 Installation
 ------------
 
+To install the latest version run:
+
 .. code-block:: bash
 
     pip install git+https://github.com/edunext/tutor-contrib-codejail
 
+You can install a specific version by adding the tag at the end, e.g.: ``#v17.0.0``.
```

Comment:
What do you think about this?
```suggestion
You can install a specific version by adding the tag, branch, or commit:

.. code-block:: bash

    pip install git+https://github.com/edunext/tutor-contrib-codejail@v17.0.0
```
https://pip.pypa.io/en/latest/cli/pip_install/#examples